CMakeLists.txt

```diff
@@ -69,18 +71,8 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${HDF5_VFD_GDS_CMAKE_MODULE_PATH})
 #------------------------------------------------------------------------------
 find_package (CUDA REQUIRED)
 
-# Set Cufile installation directory to Cuda installation directory by default
-set (HDF5_VFD_GDS_CUFILE_DIR ${CUDA_TOOLKIT_ROOT_DIR} CACHE PATH "Cufile installation directory for Nvidia GDS support")
-
 # Try to locate cufile library
-find_library (HDF5_VFD_GDS_CUFILE_LIB
-  NAMES
-    cufile
-  HINTS
-    "${HDF5_VFD_GDS_CUFILE_DIR}/lib"
-    "${HDF5_VFD_GDS_CUFILE_DIR}/lib64"
-  REQUIRED
-)
+find_package(CUDAToolKit REQUIRED)
 
 set(HDF5_VFD_GDS_EXT_INCLUDE_DEPENDENCIES
   ${HDF5_VFD_GDS_EXT_INCLUDE_DEPENDENCIES}
```

Comment:
Github doesn't let me add suggestions to the lines below, but swap these:
```diff
- CUDA_INCLUDE_DIRS
+ CUDAToolkit_INCLUDE_DIRS
```
and get rid of these variables
```diff
- CUDA_LIBRARIES
- HDF5_VFD_GDS_CUFILE_LIB
```

Reply:
Done, thanks!